x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -54,6 +54,7 @@ export interface CloudSetup {
   deploymentUrl?: string;
   profileUrl?: string;
   organizationUrl?: string;
+  snapshotsUrl?: string;
```

Comment:
nit: We've made a change to the `setup` contract and need to update the cloud plugin tests too.
Considering that we want to track consumption of the url, we should at the very least have a test in place to make sure it is there.

Reply:
Great catch! Will do.